### ISSUE TITLE
Generate the JavaScript documentation

### ISSUE THREAD

**Issue by imobachgs** [yast/d-installer#155]
Although we document the JavaScript code with [JSDoc](https://jsdoc.app/), we are not building that documentation. If we want to build the docs, we need to run jsdoc manually. And what is even worse, the format of our documentation might be wrong.

Ideally, we should define a new [script](https://github.com/joseivanlopez/d-installer/blob/d42c1d19de54a9059a6a89e9135767f3da906334/web/package.json#L7-L13) in the `package.json` to generate the documentation.

## Acceptance criteria

- [x] Running a single command generates the documentation
- [ ] Formatting issues are fixed (_mvidner_: which issues? Errors in tag syntax, reported by jsdoc? Currently we have none.)
- [x] The documentation is generated as part of our CI
- [ ] Update tasks documentation (_mvidner_: we're not sure what we meant with this :-/ anyway, added a short note to web/README.md )

**Comment by dgdavid** [comment]
Notes: Some weeks ago I was playing a bit with related things. Namely, 

* JSDoc theme/templates
  - [docdash](https://github.com/clenemt/docdash) a JSDoc 3 template inspired in (former?) [lodash documentation website](https://lodash.com/docs/4.17.15). See [available options](https://github.com/clenemt/docdash#options)
  - [minami](https://github.com/nijikokun/minami), another clean and minimal JSDoc theme/template
* Other tools probably out of the scope at this moment
  - [Docusaurus.io](https://docusaurus.io) - _beautiful documentation site in no time_. See [showcases](https://docusaurus.io/showcase), specially the [Ionic documentation](https://ionicframework.com/docs/) and [Ionic components documentation](https://ionicframework.com/docs/components)
  - [docz](https://www.docz.site/docs/introduction)
  - [React Styleguidist](https://github.com/styleguidist/react-styleguidist)
  -  [Storybook](https://storybook.js.org/) and similar tools ([Ladle](https://ladle.dev/docs/), [Vitebook](https://github.com/vitebook/vitebook), ).Out of scope at this moment, I guess.

**Comment by mvidner** [comment]
Fixed in #186.

```sh
$ cd web
$ npm run jsdoc

> jsdoc
> npx jsdoc -c jsdoc.conf.json && echo Tip: && echo xdg-open jsdoc.out/index.html

Tip:
xdg-open jsdoc.out/index.html
$ xdg-open jsdoc.out/index.html
```
